src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
-}
+  const reverse = {};
+  const entries = Object.entries(robot);
+
+  for (const object of entries) {
+    const value = object[1];
+    const key = object[0];
```

Comment:
You can simplify this by using the destructive assignment:
```suggestion
  for (const [key, value] of entries) {
```

Reply:
thanks for the advice, it's really easier and better